dbschema/ethno-art.esdl

```diff
@@ -0,0 +1,7 @@
+module default {
+  type EthnoArt extending Mixin::Named, Producible {
```

Comment:
I'd probably put mixins after "stronger"(?) parents.
```suggestion
  type EthnoArt extending Producible, Mixin::Named {
```

Reply:
Yeah, I can see that.  Sounds like a fine standard.  🤪